{
  "id": "EUVD-2025-35952",
  "enisaUuid": "6cbb813f-6abc-3977-81eb-6b0da48e3200",
  "description": "GitLab has remediated an issue in GitLab CE/EE affecting all versions from 11.7 before 18.3.5, 18.4 before 18.4.3, and 18.5 before 18.5.1 that could have allowed an unauthenticated attacker to create a denial of service condition by uploading large files to specific API endpoints.",
  "datePublished": "Oct 27, 2025, 12:05:24 AM",
  "dateUpdated": "Oct 28, 2025, 2:59:56 PM",
  "baseScore": 6.5,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
  "references": [
    "https://nvd.nist.gov/vuln/detail/CVE-2025-11974",
    "https://about.gitlab.com/releases/2025/10/22/patch-release-gitlab-18-5-1-released",
    "https://gitlab.com/gitlab-org/gitlab/-/issues/571761",
    "https://about.gitlab.com/releases/2025/10/22/patch-release-gitlab-18-5-1-released/"
  ],
  "aliases": [
    "CVE-2025-11974",
    "GHSA-rcvw-fc36-wjhj"
  ],
  "assigner": "GitLab",
  "epss": 0.03,
  "enisaIdProduct": [
    {
      "id": "34e7e7b2-8f36-35f4-aea5-1cf0c1ed73af",
      "product": {
        "name": "GitLab"
      },
      "product_version": "11.7 <18.3.5"
    },
    {
      "id": "dd077687-b475-35c0-b6b4-d3e67c615b25",
      "product": {
        "name": "GitLab"
      },
      "product_version": "18.4 <18.4.3"
    },
    {
      "id": "df5fecf1-be51-3a96-bfa7-fcbca16ba82c",
      "product": {
        "name": "GitLab"
      },
      "product_version": "18.5 <18.5.1"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "ab26d9e7-882f-34c5-8aae-ccc8feb8aa11",
      "vendor": {
        "name": "GitLab"
      }
    }
  ],
  "cve": "CVE-2025-11974"
}